main.py

```diff
@@ -61,3 +64,22 @@ def run_bash_javascript(input_code):
         out, error = process.communicate()
         string_output = out.decode('utf-8')
         return json.loads(string_output)
+
+"""
+Add semicolons to the end of java lines
+"""
+def java_linter(output_code):
+
+    if '\n' not in output_code:
```

Comment:
The if negative pattern is a little confusing - maybe add a comment or reverse the pattern

Reply:
Okay